src/ert/substitution_list.py

```diff
@@ -50,7 +50,11 @@ def from_dict(config_dict) -> SubstitutionList:
         return subst_list
 
     def substitute(
-        self, to_substitute: str, context: str = "", max_iterations: int = 1000
+        self,
+        to_substitute: str,
+        context: str = "",
+        max_iterations: int = 1000,
+        warn_max_iter=True,
```

Comment:
Github complains here:
`Function is missing a type annotation for one or more arguments`
This is likely the warn_max_iter missing `bool` annotation.